.github/workflows/auto-label.yml

```
@@ -0,0 +1,23 @@
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Label PR based on files changed
+        uses: actions/labeler@v5
+        with:
+          sync-labels: true
+        with:
```

actionlint (1.7.12)

[error] key "with" is duplicated in element of "steps" section. previously defined at line:20,col:9

(syntax-check)